src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +369,29 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal JObject GetHostJson()
+        {
+            var defaultJObject = new JObject();
+            var scriptPath = _applicationHostOptions.CurrentValue?.ScriptPath;
+            string hostFilePath = Path.Combine(scriptPath, ScriptConstants.HostMetadataFileName);
+            try
+            {
+                _logger.LogInformation("Reading host.json for SyncTrigger.");
+                string json = File.ReadAllText(scriptPath);
+                return JObject.Parse(json);
+            }
+            catch (JsonException ex)
+            {
+                _logger.LogWarning($"Unable to parse host configuration file '{scriptPath}'. : {ex.Message} : {ex.StackTrace}");
+                return defaultJObject;
+            }
+            catch (Exception ex) when (ex is FileNotFoundException || ex is DirectoryNotFoundException)
+            {
+                _logger.LogWarning($"Unable to find host configuration file '{scriptPath}'. : {ex.Message} : {ex.StackTrace}");
+                return defaultJObject;
```

Comment:
Just return a new JObject here and remove line 374.
```suggestion
                return new JObject();
```

Reply:
I think we have one more place to need to do new JObject, so that I though to remove the duplication.

Reply:
I update the PR according to your practice. How about this? @cgillum 
It looks CI fails because of the testing for SyncTrigger. I'll check the test tomorrow morning.